app/Http/Controllers/WishListController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use App\Models\WishList;
+use Auth;
+use Illuminate\Http\Request;
+
+class WishListController extends Controller
+{
+    public function index()
```

Comment:
as 'index' name suggest it should show all the wishes.
but it seems you have made it in the show() method.